lib/mysql2/client.rb

```diff
@@ -47,7 +47,7 @@ def initialize(opts = {})
       self.charset_name = opts[:encoding] || 'utf8'
 
       ssl_options = opts.values_at(:sslkey, :sslcert, :sslca, :sslcapath, :sslcipher)
-      ssl_set(*ssl_options) if ssl_options.any?
+      ssl_set(*ssl_options) if ssl_options.any? || opts.key?(:sslverify)
```

Comment:
This will turn on ssl even if the only flag is `sslverify: false` which doesn't quite make sense to me.

Reply:
I think given #879 that `ssl_set` should be called if any value of `:ssl_mode` is set, too.